plugin/src/index.ts

```diff
@@ -33,7 +33,18 @@ function addReplacingMainApplicationMetaDataItem(
 
   const mainApplication = AndroidConfig.Manifest.getMainApplicationOrThrow(manifest);
   mainApplication['meta-data'] = mainApplication['meta-data'] ?? [];
-  mainApplication['meta-data'].push(newItem);
+
+  const existingItem = mainApplication['meta-data'].find(
+    item =>
+      item.$['android:name'] === itemName &&
+      item.$['android:value'] === itemValue &&
+      item.$['tools:replace' as keyof AndroidConfig.Manifest.ManifestMetaData['$']] ===
+        'android:value',
+  );
+
+  if (!existingItem) {
```

Comment:
If it exists, we should update it with the latest value i think?

Reply:
I thought about it for a little, and yes, overwriting values put into the manifest by hand or other config plugins sounds reasonable. I updated the PR.